tasks/GPSR.tex

```diff
@@ -28,7 +28,7 @@ \subsection*{Setup}
         \begin{itemize}
             \item \textbf{Operators:} A \emph{Professional Operator} (the referee) commands the robot to execute a task.
             Optionally, commands can be issued by a \emph{Non-Expert Operator}, i.e. a person from the audience with no robotics background.
-            In this case, the referee gives the command to the non-expert operator, who will then issue it to the robot (rephrasing is allowed).
+            In this case, the referee gives the goal of the command to the non-expert operator, who will then issue it to the robot in their own words. For example, the generated command might be "Bring me a coke from the kitchen." then the non-expert operator will be told "The robot should bring you a coke, which is found in the kitchen.", who then tells the robot "I want a coke. Go to the kitchen and get me one."
```

Comment:
Again, why is the diff marked red here?

Reply:
This one is weirder. Maybe changing the quotation marks work here as well